core/store/presenters/presenters.go

```diff
@@ -568,3 +568,45 @@ func (t *Tx) SetID(hex string) error {
 	t.Hash = common.HexToHash(hex)
 	return nil
 }
+
+// ExternalInitiator includes initiator and authentication details.
+type ExternalInitiator struct {
```

Comment:
Moving to presenters: nice, having the generic name `ExternalInitiator` concerns me though, as it may suggest for people to use for /show endpoints and the like. I gave this the name `ExternalInitiatorAuthentication` initially as it includes secret credentials that can only be used at creation time and never again.

Reply:
Updated to ExternalInitiatorAuthentication